functions/graphql/schema/simulator.js

```diff
@@ -62,9 +62,9 @@ module.exports.resolver = {
     }
   },
   Mutation: {
-    simulatorCreate: async (rootQuery, { name }, context) => {
+    simulatorCreate: async (rootQuery, { name, stations }, context) => {
```

Comment:
I mentioned this above, but to reiterate - we don't need to have `stations` as an argument when creating simulators. The stations are passed by Thorium when a flight record is created. We need to be able to access stations when resolving a flight record, but in Space EdVentures a simulator stored in the database has no concept of stations.